homeassistant/components/tradfri/light.py

```diff
@@ -22,20 +23,15 @@
 import homeassistant.util.color as color_util
 
 from .base_class import TradfriBaseEntity
-from .const import (
-    ATTR_DIMMER,
-    ATTR_HUE,
-    ATTR_SAT,
-    ATTR_TRANSITION_TIME,
-    CONF_GATEWAY_ID,
-    COORDINATOR,
-    COORDINATOR_LIST,
-    DOMAIN,
-    KEY_API,
-    SUPPORTED_LIGHT_FEATURES,
-)
+from .const import CONF_GATEWAY_ID, COORDINATOR, COORDINATOR_LIST, DOMAIN, KEY_API
 from .coordinator import TradfriDeviceDataUpdateCoordinator
 
+ATTR_DIMMER = "dimmer"
+ATTR_HUE = "hue"
+ATTR_SAT = "saturation"
+ATTR_TRANSITION_TIME = "transition_time"
```

Comment:
These four constants should preferably just be replaced by using keyword arguments inline in the calls where needed.